mergify_engine/actions/merge/helpers.py

```diff
@@ -73,21 +73,26 @@ def merge_report(ctxt, strict):
     return conclusion, title, summary
 
 
-def get_wait_for_ci_report(ctxt, rule=None, missing_conditions=None):
-    summary = "The pull request has been automatically updated to follow its base branch and will be merged soon."
+def get_strict_status(ctxt, rule=None, missing_conditions=None, need_update=False):
+    if need_update:
+        title = "Base branch will be updated soon"
+        summary = "The pull request base branch will be updated soon, and then merged."
+    else:
+        title = "Base branch updates done"
```

Comment:
```suggestion
        title = "Base branch update done"
```
I think it's a single update.